Expose DD_SERVERLESS_LOGS_ENABLED as env var (#48)
DataDog/datadog-cdk-constructs: README.md

| `apiKey` | `api_key` | Datadog API Key, only needed when `flushMetricsToLogs` is `false` or `extensionLayerVersion` is set. For more information about getting a Datadog API key, see the [API key documentation][8]. |
| `apiKmsKey` | `api_kms_key` | Datadog API Key encrypted using KMS. Use this parameter in place of `apiKey` when `flushMetricsToLogs` is `false` or `extensionLayerVersion` is set, and you are using KMS encryption. |
| `enableDatadogTracing` | `enable_datadog_tracing` | Enable Datadog tracing on your Lambda functions. Defaults to `true`. |

> | `enableDatadogLogs` | `enable_datadog_logs` | Enable Datadog logs on your Lambda functions. Defaults to `true`. |

Reviewer: Per my comment on your [other PR](https://github.com/DataDog/serverless-plugin-datadog/pull/146), I don't think this will apply when using the Datadog Forwarder instead of the Datadog Lambda Extension. Let me know if you want more info about this! 🙂

Author: Thanks! I don't think I'm getting github notifications via email yet, so I didn't see that comment yesterday

> | `enableDatadogLogs` | `enable_datadog_logs` | Enable Datadog logs on your Lambda functions. Note: This applies only to integrations using the Datadog Extension. This setting has no effect when used in conjunction with the Log Forwarder. Defaults to `true`. |

Reviewer: ```suggestion
| `enableDatadogLogs` | `enable_datadog_logs` | Send Lambda function logs to Datadog via the Datadog Lambda Extension.  Defaults to `true`. Note: This setting has no effect on logs sent via the Datadog Forwarder. |
```
